transformer/transformer.go

```diff
@@ -85,3 +87,39 @@ func (t *Transform) GetTransactionResources(payment data.PaymentResponse,
 
 	return paymentTransactionsResources, nil
 }
+
+// GetTransactionResources transforms payment data into payment transaction resource entities
+func (t *Transform) GetRefundResource(payment data.PaymentResponse,
+	refund data.RefundResource,
+	paymentId string) (models.RefundResourceDao, error) {
+
+	refundResource := models.RefundResourceDao{}
+
+	refundDate, err := time.Parse(time.RFC3339Nano, refund.CreatedAt)
+	if err != nil {
+		return refundResource, err
+	}
+
+	productMap, err := config.GetProductMap()
+	if err != nil {
+		return refundResource, err
+	}
+
+	refundResource = models.RefundResourceDao{
+		TransactionID:     "X" + refund.RefundId,
```

Comment:
Please confirm with Stephen if the X should be prepended

Reply:
I asked Stephen, he will return to me with the info in an hour.

Reply:
Stephen confirmed the X should be there